docker/README.md

```diff
@@ -5,11 +5,19 @@ You may build this via
 docker build --tag aerogear.org .
 ```
 
+To run the container:
+
+```
+./run.sh
+```
+from this directory (does not work if you run it from a different directory).
+
+
```

Comment:
Here we should insert a new section "Development" or something similar.  I don't feel that the flow from "here is a script" to "here is a long docker command" flows particularly well.

Optionally run.sh could be edited to take a --mount option which will mount the aerogear directory it is currently in.